src/backend/media-route-handlers.ts

```diff
@@ -1,8 +1,21 @@
+import { createId } from '@paralleldrive/cuid2'
+import * as t from 'io-ts'
+import {
+  PutObjectCommand,
+  PutObjectCommandInput,
+  S3Client,
+} from '@aws-sdk/client-s3'
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner'
 import { createProxyMiddleware } from 'http-proxy-middleware'
 import { readEnvVariable } from './read-env-variable'
+import type { Request, Response } from 'express'
 
-const target = new URL(readEnvVariable('S3_ENDPOINT'))
-target.pathname = readEnvVariable('BUCKET_NAME')
+const endpoint = readEnvVariable('S3_ENDPOINT')
+const bucketName = readEnvVariable('BUCKET_NAME')
+const region = readEnvVariable('BUCKET_REGION')
```

Comment:
Hmm. I agree that from a security standpoint that is not super important.
But I'm not sure about the DX point:
Since we need the access keys from env any way it feels okay for me to just get the while bucket info from `.env`.
And the code is even a bit cleaner since I don't have any conditionals. 

Slight preference for the way it is but also don't mind hardcoding.